### ISSUE TITLE
Jaeger UI <cannot-find-trace-name>

### ISSUE THREAD

**Issue by imatefx** [moleculerjs/moleculer-metrics#2]
If rest API request is successful I get correct trace name

![image](https://user-images.githubusercontent.com/3025904/37775483-199b2e10-2e09-11e8-9453-9117fba67744.png)

If API request is error the trace name is " < cannot-find-trace-name > " also the api service is not listed

![image](https://user-images.githubusercontent.com/3025904/37775861-144ac30c-2e0a-11e8-8f25-29f2476f406c.png)

Is it related to setting parentCtx on moleculer ``context.js`` file?

Below is the metric data received on ``makePayload`` Function for a failed api request

```
{ id: '9cc98685-a241-4ce0-b4d6-a38867a0bc57',
  requestID: 'f92f5c36-0518-4808-b63d-cdeaa83ccbd5',
  level: 2,
  startTime: 1521729207551,
  endTime: 1521729208835.64,
  duration: 1284.639944,
  remoteCall: false,
  fromCache: false,
  params: { dbName: 'm1213' },
  meta: { username: undefined },
  action: { name: 'v1.db.create' },
  parent: 'f92f5c36-0518-4808-b63d-cdeaa83ccbd5',
  nodeID: 'win-n53jq-11308',
  error:
   { name: 'MoleculerError',
     code: 403,
     type: 'DB_CREATE_FAILED',
     message: 'Unable to create Db' } }
```

```
{ id: '9cc98685-a241-4ce0-b4d6-a38867a0bc57',
  requestID: 'f92f5c36-0518-4808-b63d-cdeaa83ccbd5',
  level: 2,
  startTime: 1521729207551,
  endTime: 1521729208896.2466,
  duration: 1345.246633,
  remoteCall: false,
  fromCache: false,
  params: { dbName: 'm1213' },
  meta: { username: undefined },
  action: { name: 'v1.db.create' },
  parent: 'f92f5c36-0518-4808-b63d-cdeaa83ccbd5',
  nodeID: 'win-n53jq-11308' }
```

**Comment by icebob** [comment]
Thanks, I will check it.

**Comment by icebob** [comment]
It's fixed in `moleculer-web@0.7.0-beta2`

![image](https://user-images.githubusercontent.com/306521/37878993-47dc349e-3071-11e8-89f1-e21b4bb47278.png)
